CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 Latest
 ------
-
+* Fixed `aiaprep` to return properly sized map.
```

Comment:
So full field of view and centered on the Sun?  What is the problem with
submaps?


On Mon, May 16, 2016 at 12:48 PM, Larry Manley <notifications@github.com>
wrote:

> In CHANGELOG.md
> <https://github.com/sunpy/sunpy/pull/1756#discussion_r63384608>:
>
> > @@ -1,6 +1,6 @@
> >  Latest
> >  ------
> > -
> > +* Fixed `aiaprep` to return properly sized map.
>
> @wafels <https://github.com/wafels> It will also work for full-disk
> images that have been resized (2048^2, 1024^2, etc.), but not submaps.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/sunpy/sunpy/pull/1756/files/e4299af358aa0ea61d676789839b0feec7754479#r63384608>
>